requirements.txt

```diff
@@ -40,5 +40,5 @@ deprecated~=1.2
 jinja2~=3.1, >=3.1.3
 orjson>=3.9.15, <4
 # mlrun pipeline adapters
-mlrun-pipelines-kfp-common~=0.1.3
-mlrun-pipelines-kfp-v1-8~=0.1.3
+mlrun-pipelines-kfp-common~=0.1.5
+mlrun-pipelines-kfp-v1-8~=0.1.5
```

Comment:
unfortunately - you can't update them before the bump version is merged & released - so revert these changes and once merged - I will trigger release